### DISCUSSION TITLE
Is there a procedure for increasing the API request rate limit on testnet like on mainnet?

### DISCUSSION THREAD

**Discussion by aashborn** [ProjectOpenSea/opensea-js#1377] [Q&A] [answered]
Does the procedure for increasing the rate limit on mainnet also apply to the tesnet?

Is there any way to make this change on testnet?

Thanks

**Comment by ryanio** [comment, marked as answer]
No, mainnet and testnet keys are separate. Testnet does not require an api key. You can try submitting a support request if you want a testnet key with higher rates.

**Reply by aashborn** [reply]
Okay, we will try, thanks!
